CHANGELOG.md

```diff
@@ -1,10 +1,19 @@
 # [unreleased](https://github.com/jamesxX/valakyrie/releases/tags/)
 ## Added
+- `Boolean` validation type
+- `Content` validation type. Also accepts strings which are coerced into content types.
+- `Color` validation type.
+- `Optional` validation type. If a schema yields a validation error, the error is suppressed and the returned value is 'auto'
+- `Choice` validation type. Tested value must be contained within the listed choices.
 
 ## Removed
 
 ## Changed
-
+- fixed error in documentation for string type (previously read that it worked on numbers).
+- Dictionaries that don't have a member that is present in the schema no longer produce an error outside of strict contexts.
+- **(Potentially breaking)** Dictionaries default to empty dictionaries rather than none.
```

Comment:
The change that motivated this is something I wanted to suggest before, but always forgot to do so. This is _definitely_ a breaking change, as most usages of `z.dictioanry` now need additional parentheses.

Reply:
Thanks. I'm considering splitting the changelog into a changelog and a migration guide (or having a migration guide as part of the changelog, I don't know yet).

Reply:
Migration guide or not, it should definitely be marked as a breaking change, not a potentially breaking change.

Reply:
done, [d5e26ae](https://github.com/JamesxX/valkyrie/pull/5/commits/d5e26ae602eadfa3e73ff13f5c9a86b203689151)